packages/manager/modules/sms/package.json

```diff
@@ -32,10 +32,12 @@
     "@ovh-ux/ng-ovh-telecom-universe-components": "^5.0.0 || ^6.0.0",
     "@ovh-ux/ng-ovh-ui-confirm-modal": "^2.0.0",
     "@ovh-ux/ng-pagination-front": "^10.0.0",
+    "@ovh-ux/ng-ui-router-layout": "^4.0.0",
     "@ovh-ux/ui-kit": "^4.1.12",
     "@uirouter/angularjs": "^1.0.15",
     "angular": "^1.7.5",
     "angular-messages": "^1.7.6",
+    "angular-ui-bootstrap": "~1.3.3",
```

Comment:
Not really a peer deps for the `@ovh-ux/manager-sms` module (not imported), but it's a peer deps of one peer dep: `@ovh-ux/ng-ui-router-layout`.

Reply:
thanks for explanations about AngularJS things ! i've done on last commit :)